src/gcov.rs

```diff
@@ -51,21 +20,16 @@ pub fn run_gcov(gcno_path: &PathBuf, branch_enabled: bool, working_dir: &PathBuf
     let status = command
         .arg(gcno_path)
         .arg("-i") // Generate intermediate gcov format, faster to parse.
-        .current_dir(working_dir)
-        .stdout(Stdio::null())
-        .stderr(Stdio::null());
+        .current_dir(working_dir);
 
-    /*if cfg!(unix) {
-        status.spawn()
-              .expect("Failed to execute gcov process");
-    } else {*/
-    let status = status.status().expect("Failed to execute gcov process");
-    assert!(
-        status.success(),
-        "gcov wasn't successfully executed on {}",
-        gcno_path.display()
-    );
-    //}
+    let output = status.output().expect("Failed to execute gcov process");
+
+    if !output.status.success() {
+        eprintln!("gcov stdout: {}", String::from_utf8_lossy(&output.stdout));
+        eprintln!("gcov stderr: {}", String::from_utf8_lossy(&output.stderr));
+
+        panic!("gcov wasn't successfully executed on {}", gcno_path.display());
```

Comment:
maybe improve the wording with 
gcov execution failed on {}

Reply:
Good idea, done with dba3edcf581a6f46e13d4ec4d2086809b87fc0d6.